.github/ISSUE_TEMPLATE/report_a_bug.yml

```diff
@@ -37,14 +37,6 @@ body:
     validations:
```

Comment:
Maybe include `, including the behavior you expected to see.` in the description ob the above field? So both fields become integrated.

Reply:
Done